extras/devices/LVFS/tests/test_LVFS.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import json
+from devices import *
+from unittest.mock import patch
+from devices.LVFS.tests import fwupd_data
+
+device_info = json.loads(fwupd_data.get_device)
+device_results = json.loads(fwupd_data.get_results)
+
+
+class TestLVFSDevice(unittest.TestCase):
+    def mock_run_ssh(*args, **kwargs):
+        global device_info
+        global device_results
+        if args[1] == "sudo fwupdmgr refresh --force":
+            return 0, "", ""
+        elif args[1] == "sudo fwupdmgr get-devices --json":
+            return 0, json.dumps(device_info), ""
+        elif "sudo fwupdmgr upgrade" in args[1]:
+            return 0, "", ""
+        elif "sudo fwupdmgr download" in args[1]:
+            return 0, "", ""
+        elif "sudo fwupdmgr install" in args[1]:
+            return 0, "", ""
+        elif "sudo fwupdmgr get-results" in args[1]:
+            return 0, json.dumps(device_results), ""
```

Comment:
Is it possible to simplify this logic and use less elif statements?

Reply:
Done. Simplified as per @plars 's suggestion below.